src/views/blocks/tickets.php

```diff
@@ -72,7 +72,7 @@
 	data-provider-id="<?php echo esc_attr( $provider->orm_provider ); ?>"
 	novalidate
 >
-	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title"><?php esc_html_e( 'Tickets', 'event-tickets' ); ?></h2>
+	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title"><?php esc_html_e( tribe_get_ticket_label_plural( 'event-tickets' ), 'event-tickets' ); ?></h2>
```

Comment:
No need to translate this, as it already should be translated:

```suggestion
	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title"><?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?></h2>
```